particle/webpack.md

```diff
@@ -1,7 +1,33 @@
 # Webpack
 
+[Webpack](https://webpack.js.org/) is a bundling tool to handle the various assets of the project in a build step. It accepts one or more entry points and then tracks their various dependencies recursively to assemble the dependency graph. This dependency chain allows us to readily include files to be emitted to the built assets of the project. The Webpack configuration in Particle has been set up to handle commonly used file types such as: Twig, Sass, JavaScript, TypeScript, SVGs, image assets, and more.
```

Comment:
Consider linking to official docs when using webpack jargon. E.g, entry points could be pointed to: https://webpack.js.org/concepts/entry-points/

Reply:
Good call. Updates incoming.

Reply:
Added.